TeaDiary/Scripts/App/TeaCommands.js

```diff
@@ -0,0 +1,46 @@
+﻿function TeaCommands() {
+    var self = this;
+    self.delete = (id, successCallback, errorCallback) => {
+        popupConfirmation(
```

Comment:
"popup" is a noun. Method names must be verbs or verb phrases. Think `showPopupConfirmation`.